### PR TITLE
make support-frontend start up 15 seconds quicker locally

### PR DESCRIPTION
**Pull request by johnduffell**
## What are you doing in this PR?

This PR uses the later version of simple-configuration which bypasses the slow code in dev - https://github.com/guardian/simple-configuration/pull/25

https://trello.com/c/XHdiaOO5/3877-make-support-frontend-play-app-16-seconds-quicker-to-start

## Why are you doing this?

it's frustrating waiting 2x7+ seconds every time you make a tiny scala change in support-frontend.

## Screenshots
before
image

After
image

### CODE REVIEW

File: support-frontend/app/wiring/AppLoader.scala [hunk 1]
```diff
@@ -18,25 +19,28 @@ class AppLoader extends ApplicationLoader with StrictLogging {
...
-  private def getParameterStoreConfig(initialConfiguration: Configuration): Configuration = {
-    val identity = AppIdentity.whoAmI(defaultAppName = "support-frontend", CredentialsProvider)
-    val loadedConfig = ConfigurationLoader.load(identity, CredentialsProvider) {
-      case AwsIdentity(app, stack, stage, _) =>
-        val privateConfig = SSMConfigurationLocation(s"/$stack/$app/$stage")
-        val publicConfig = ResourceConfigurationLocation(s"$stage.public.conf")
+  private def getParameterStoreConfig(isDev: Boolean): Try[Configuration] = {
+    for {
+      identity <- if (isDev)
+        Success(DevIdentity("support-frontend"))
```
Comment (rupertbates) [inline support-frontend/app/wiring/AppLoader.scala:25]: Don't we need to load the local private config as in the case on line 35?

Comment (rupertbates) [reply support-frontend/app/wiring/AppLoader.scala:25]: Oh I see how it works now, it was a bit tricky to follow in Github

**Review by rupertbates — APPROVED** [review]
👍

File: support-frontend/app/wiring/AppLoader.scala (hunk 1 above)
Comment (johnduffell) [reply support-frontend/app/wiring/AppLoader.scala:25]: yeah it is a bit hard to follow, there may be further imrpovmenets possible!

File: support-frontend/build.sbt [hunk 2]
```diff
@@ -11,7 +11,7 @@ testOptions in Test := Seq(Tests.Filter(unitTestFilter))
 
 libraryDependencies ++= Seq(
   "com.typesafe" % "config" % "1.3.2",
-  "com.gu" %% "simple-configuration-ssm" % "1.5.6",
+  "com.gu" %% "simple-configuration-ssm" % "1.5.7-SNAPSHOT",
```
[the remark this answers is not in the document]
Comment (johnduffell) [reply support-frontend/build.sbt:14]: this is now superseded by https://github.com/guardian/simple-configuration/pull/28

**Comment by johnduffell** [comment]
tested in CODE and starts up okay, also monthly contribution checked out OK
